tasks/input_sql_file.yml

```diff
@@ -0,0 +1,35 @@
+- name: Verify if postgresql_password is set
+  assert:
+    that: postgresql_password is defined
```

Comment:
since it is defined in defaults/main.yml, it will always be defined - so
`that: not postgresql_password is none`

Reply:
`not postgresql_password is none` would work if `postgresql_password: null` is set in defaults/main.yml. And it is a good practice to set all variables that user can provide in defaults/main.yml.